### DISCUSSION TITLE
Home base alarm panel

### DISCUSSION THREAD

**Discussion by MLux72** [fuatakgun/eufy_security#250] [Q&A] [answered]
On the home base alarm panel I have to switch firt to disarm and than I can select an other state, home, away or sleep.
Is it possible to directly switch from home to away, without first disarm?

**Comment by Brouwertje** [comment, marked as answer]
I dont think so. You could make a script for arming away and home mode and make your own buttons for it.

**Comment by fuatakgun** [comment]
As it is called out, this is a design decision by home assistant UI, you can rely on service calls.

**Reply by MLux72** [reply]
Oké thanks
